README.md

```diff
@@ -130,17 +130,26 @@ A non-exhaustive collection of third-party clients and mods for Discord.
 | [Switchcord](https://github.com/vbe0201/switchcord) | An unofficial Discord client for the Nintendo Switch console. | [![C++][C++-Badge]][C++-Url] | 🔴 Discontinued |
 | [3DiScord](https://github.com/cheuble/3DiScord) | A Discord client for the Nintendo 3DS | [![C++][C++-Badge]][C++-Url] | ⛔ Discontinued. will get you banned |
 
-## Other
+## Other clients
```

Comment:
Since Arcscord isn't a client, the category shouldn't say "clients". We might add a separate category for the libraries in the future.

Reply:
Is it not a client library?

Reply:
Yes, it's true, I am sometimes stupid. I might also move discord-aos, crocodile and discord-ppc to the reimplementation category.